python/flatbuffers/util.py

```diff
@@ -20,6 +20,13 @@ def GetSizePrefix(buf, offset):
 	"""Extract the size prefix from a buffer."""
 	return encode.Get(packer.int32, buf, offset)
 
+def GetBufferIdentifier(buf, offset, size_prefixed = False):
+        """Extract the file_identifier from a buffer"""
+        offset+=(number_types.UOffsetTFlags.bytewidth if size_prefixed else 0) + number_types.UOffsetTFlags.bytewidth # offset sizeof(root table offset) + sizeof(size prefix) if present   
+        return buf[offset:offset+encode.FILE_IDENTIFIER_LENGTH]
+def BufferHasIdentifier(buf, offset, file_identifier, size_prefixed = False):
+        return GetBufferIdentifier(buf, offset, size_prefixed)==file_identifier
```

Comment:
When calling functions with keyword arguments: please provide the arguments as keyword arguments, not positional arguments.

Reply:
I added this test

Reply:
@rw i found a couple more occureces of this in different files, including the code generated by idl_gen_python.cpp. I have fixed them.